noise2same/backbone/decode_head/decode_head.py

```diff
@@ -0,0 +1,149 @@
+from abc import ABCMeta, abstractmethod
+
+import torch
+import torch.nn as nn
+from noise2same.backbone.ops import resize
+
+
+class BaseDecodeHead(nn.Module, metaclass=ABCMeta):
```

Comment:
if these file are copypaste it's best to reference the source (and check license)